README.md

```diff
@@ -6,23 +6,27 @@
 <a href="https://github.com/camaraproject/QualityOnDemand/blob/main/documentation/LICENSE.APACHE2.0" title="License"><img src="https://img.shields.io/badge/License-Apache%202.0-green.svg?style=plastic"></a>
 
 # QualityOnDemand
+
 Repository to describe, develop, document and test the QualityOnDemand API family
 
 ## Scope
-* Service APIs for “Quality on Demand” (see APIBacklog.md)  
+
+* Service APIs for “Quality on Demand” (see APIBacklog.md)
 * It provides the customer with the ability to:  
-  * set quality for a mobile connection (e.g. required latency, jitter, bit rate)  
+  * set quality for access network connections (e.g. mobile device connection or fixed access between a home gateway and the service providers gateway router)  
   * get notification if network cannot fulfill  
-  * NOTE: The scope of this API family should be limited (at least at a first stage) to 4G and 5G.  
-* Describe, develop, document and test the APIs (with 1-2 Telcos)  
+  * NOTE: The scope of this API family should be limited the networks between the users's device (e.g. mobile phone or home gateway) and the gateway from the operator's network to other networks.
```

Comment:
```suggestion
  * NOTE: The scope of this API family should be limited to the networks between the users's device (e.g. mobile phone or home gateway) and the gateway from the operator's network to other networks.
```

Actually this "limitation" isn't any more necessary, as the same scope description is also already in line 16.